### ISSUE TITLE
Initial staging environment

### ISSUE THREAD

**Issue by ChristianUlbrich** [zalari/tcq#2]
After #1 has been completed, we want to run the "old" stack in a staging setup, i.e. simply deploy it to an environment, that can be publicly tinkered with.

# Initial Goals
There is a _semi-public_ instance running for the "server" and the "document storage", it is reachable via a valid and secure DNS name.

## Acceptance Criteria
* [ ] deployment is automated in such a way, that _manually_ the stack can be completely wiped clean and re-deployed
* [ ] deployment for the "server" should be via a Kubernetes _deployment_ with a configured Kubernetes _service_
* [ ] we have configured a running Kubernetes host somewhere (internally is fine for the time being)
* [ ] there is an additional repository that is public containing the code for the (internal) infrastructure; there shall be no credentials inside of it, i.e. everything must be configurable via the environment as well
* [ ] we manually configure our internal infrastructure for proper DNS setup (edge-traefik -> to internal traefik...)

## Out of scope
* DNS will not be automated, but supplied via the _config_. DNS has to be setup manually.
* Azure document storage will be supplied and created manually and has also to be _configured_.

**Comment by ChristianUlbrich** [comment]
Done with #8
